.github/workflows/codeql-analysis.yml

```diff
@@ -17,7 +17,7 @@ jobs:
     permissions:
       security-events: write  # for github/codeql-action/autobuild to send a status report
     name: Analyze
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-latest
```

Comment:
Any reason this isn't using ubuntu 22.04?

Reply:
Used the same as whats there in `webapp`

https://github.com/mattermost/mattermost/blob/ea4ab9aa90fe1557f1f3a292c4c992eaded3b818/.github/workflows/codeql-analysis.yml#L19